.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Report bugs or errors
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Run code '...'
+2. Expect this error '....'
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Environment, if relevant:**
+ - OS: [e.g. iOS]
```

Comment:
I don't think iOS is a good example since as far as I know we don't support iOS, maybe better examples would be Ubuntu 22.04, macOS 14.5, etc.